cluster_kwargs.yaml

```diff
@@ -40,6 +40,12 @@ embarrassingly_parallel_cluster:
   backend_options:
     region: "us-east-1"  # Same region as dataset
 
+# For tests/workflows/test_uber_lyft.py
+uber_lyft_cluster:
+  n_workers: 30
+  backend_options:
+    region: "us-east-1"  # Same region as dataset
```

Comment:
```suggestion
    region: "us-east-2"  # Same region as dataset
```
The dataset being used at the moment, the `s3://coiled-datasets/mrocklin/` is in `us-east-2` as most of the datasets inside `coiled-datasets`

Reply:
And `s3://coiled-datasets/uber-lyft-tlc/`?

Reply:
Same, if you have doubts about where they are in the AWS ui, once you are in the bucket/folder you can go over to properties, and see where it is. 
![Screen Shot 2023-04-03 at 7 16 11 PM](https://user-images.githubusercontent.com/7526622/229646707-7f407927-58f2-426a-8581-ac33616a53e8.png)